### DISCUSSION TITLE
ExtendClientAreaToDecorationsHint changes resize behavior

### DISCUSSION THREAD

**Discussion by wawrzenczyka** [AvaloniaUI/Avalonia#9080] [Q&A]
Hi, 
I've noticed that setting ExtendClientAreaToDecorationsHint="True" on a window changes the way resizing works. For most windows (say, Windows Explorer) you are able to resize a window starting right on the window border, and several pixels outwards. The same holds true for Avalonia apps without aforementioned setting. When ExtendClientAreaToDecorationsHint is set to True. you still are able to resize at the window border, but this time the resize cursor disappears when moving mouse outward, and instead you can start resizing a few pixels inside the window. For an example, [MusicStore tutorial app](https://github.com/AvaloniaUI/Avalonia.MusicStore) behaves like this.

This starts to become the problem when edge of the window contains elements which interact with mouse, most notably a scrollbar. When attempting to interact with a scrollbar, resize cursor appears, making it difficult/impossible to achieve.

This prompts the question - is it possible to somehow circumvent this issue? I'd like to customize/get rid of the default title bar to blend in with my application dark mode, and based on my searches on the issue I believe that in order to do that I need to set ExtendClientAreaToDecorationsHint="True" or HasSystemDecorations="False" (which gets rid of the resize completely). I think the resize behavior is caused by Avalonia handling the resize instead of the operating system, making it impossible to use the area outside the window, and to fix it in the library itself. I'd be thankful for any suggestions.

**Comment by yll690** [comment]
It's the same in WPF when you customize the title using WindowChrome. As far as I know, there is no simple solution. It's said that the Visual Studio solved this by adding four(or three) invisible window around the main window to accept the mouse event.
